modules/backend/formwidgets/recordfinder/partials/_recordfinder.htm

```diff
@@ -1,10 +1,16 @@
+<?php
```

Comment:
@LukeTowers using my application I realized that I just forgot to apply the same snippet code you've suggested for the switch widget, to the recordfinder. Now the recordfinder and switch, both has the same behaviour.